bait-files/src/main/webapp/review.html

```diff
@@ -174,16 +200,13 @@
             </div>
             <div id="address-verify">
               <p class="sub-header">Address Verification</p>
-              <label for="address-status">Verify address:</label>
-              <select name="address-status" id="address-status">
+              <label for="addressValidity">Verify address:</label>
+              <select name="addressValidity" id="addressValidity">
```

Comment:
Style guide wise, the for, name, and id attributes should be all lowercase and hyphenated: address-validity.